src/components/Dashboard/TodoList.vue

```diff
@@ -294,7 +316,13 @@
         </v-expansion-panel-header>
 
         <v-expansion-panel-content>
-          <div v-if="isTypeCorrection(task)">
+          <v-card v-if="isBcolError(task)" class="todo-list-detail bcol-todo-list-detail">
```

Comment:
I'm working on this code atm (in #3306) so you will have to make some of my changes here when you merge after me. Basically the expansion content is now a div instead of a v-card and uses some classes for styling. Your best bet will be to see what the other expansion blocks look like and copy that.

Reply:
Last person to merge will have to handle this.

Reply:
As this PR was merged before mine, I will update this.